### PR TITLE
Phone numbers are now encypted!

### PR DESCRIPTION
**Pull request by nsdeschenes**
- Phone numbers are now encrypted before storing in db
- `updateUserProfile` mutation can also update the users phone number
- All related functionality now encrypts and decrypts phone number as required
- New user types, shared and personal
  - Shared used for affiliations, i.e. lists of users in an org, or `findUserByUsername`
  - Personal used for  `findMe` query
- 100 Test coverage across the board for all newly implemented items

### CODE REVIEW

File: api-js/src/mutations/user/__tests__/send-phone-code.test.js [hunk 1]
```diff
@@ -182,7 +182,7 @@ describe('user send password reset email', () => {
 
           expect(response.errors).toEqual(error)
           expect(consoleOutput).toEqual([
-            `User attempted to send TFA text message, however no account is associated with 1.`,
+            `User attempted to send TFA text message, however no account is associated with this key 1.`,
```
Comment (Ethanljf) [inline api-js/src/mutations/user/__tests__/send-phone-code.test.js:185]: "this key 1." -- intentional?

File: api-js/src/mutations/user/__tests__/send-phone-code.test.js [hunk 2]
```diff
@@ -485,7 +485,7 @@ describe('user send password reset email', () => {
...
```
Comment (Ethanljf) [inline api-js/src/mutations/user/__tests__/send-phone-code.test.js:488]: See other comment

File: api-js/src/mutations/user/__tests__/send-phone-code.test.js (hunk 1 above)
Comment (nsdeschenes) [reply api-js/src/mutations/user/__tests__/send-phone-code.test.js:185]: `userKey: 1,`

File: api-js/src/mutations/user/__tests__/send-phone-code.test.js (hunk 2 above)
Comment (nsdeschenes) [reply api-js/src/mutations/user/__tests__/send-phone-code.test.js:488]: See above reply

**Review by Ethanljf — APPROVED** [review]

**Pull request merged**